django/core/management/color.py

```diff
@@ -14,11 +14,57 @@ def supports_color():
     Return True if the running system's terminal supports color,
     and False otherwise.
     """
-    supported_platform = sys.platform != 'win32' or 'ANSICON' in os.environ
+
+    def supported_platform():
+        """
+        Determines if the current platform will support terminal color.
+        """
+
+        if sys.platform != 'win32':
+            return True
+
+        if 'ANSICON' in os.environ:
+            return True
+
+        # Microsoft Terminal supports VT codes
+        if 'WT_SESSION' in os.environ:
+            return True
+
+        # Microsoft Visual Studio Code's built-in terminal supports colors
+        if 'TERM_PROGRAM' in os.environ and os.environ['TERM_PROGRAM'] == 'vscode':
```

Comment:
```suggestion
        if os.environ.get('TERM_PROGRAM') == 'vscode':
```

Reply:
Fixed in 2ac0a53e18244cc6d1fbafe7cce967ace3e1fd65